charts/gsp-cluster/templates/01-aws-system/aws-service-operator.yaml

```diff
@@ -0,0 +1,232 @@
+apiVersion: v1
+kind: List
```

Comment:
It's unusual to add a List resources in this context ... better to have seperate files for the resources.

Reply:
This is from upstream. We could break it up if we don't mind significantly diverging.